ffi/custom_passes.cpp

```diff
@@ -590,7 +597,7 @@ struct RefPrunePass : public FunctionPass {
      *  - true if the fan-out condition specified by `prune_raise_exit` was
      *    found, false otherwise.
      */
-    bool findFanout(CallInst *incref, SmallBBSet *decref_blocks, bool prune_raise_exit) {
+    bool findFanout(CallInst *incref, SmallBBSet &bad_blocks, SmallBBSet *decref_blocks, bool prune_raise_exit) {
```

Comment:
These parameter additions need adding to the docstring.

Reply:
fixed in fa6092b